Login

kevinlueke opened the pull request.

kevinlueke committed:
Teacher and students may now log on

kevinlueke committed:
Create login for students and teachers

kevinlueke committed:
More comments to ensure readabilty

Steve-D-Eckles commented on portal/auth.py:
```diff
@@ -0,0 +1,83 @@
+        )
+        user = cur.fetchone()
+        cur.close()
+        print(user)
```
These print statements used for debugging should probably not be included in the final code

Steve-D-Eckles commented on requirements.txt:
```diff
@@ -1,7 +1,10 @@
 attrs==19.3.0
+bcrypt==3.1.7
```
Are these changes to the requirements necessary? It doesn't seem like you ultimately decided to use bcrypt.

kevinlueke committed:
Clean code up a little

Remove print statements used for debugging from auth.py and remove becrypt from requirements.txt

kevinlueke committed:
Remove two items from requiremnts.txt
